content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Entity Framework Core
+
+## Who Should Read This?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
+
+Before beginning this tutorial, you should understand a few topics. One, this tutorial assumes a familiarity with C#. If you're new to C#, you may wish to go through the [Getting Started with C#](https://www.microsoft.com/net/tutorials/csharp/getting-started) tutorial. Two, you should have a basic knowledge of relational database concepts. Three, this tutorial will use LINQ statements. If you have not used LINQ before, the [Introducing LINQ](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) for an introduction to LINQ) tutorial can help you get up-to-speed on LINQ.
```

Comment:
Yeah, that's personal preference. I like "one, two, three" better because the tutorial states that it assumes knowledge of a few basic concepts. Unless @arynhuck or @davidmikh agree with you, I'd say you're outvoted 😛